cmd/gosec/main.go

```diff
@@ -457,6 +464,14 @@ func main() {
 
 	reportInfo := gosec.NewReportInfo(issues, metrics, errors).WithVersion(Version)
 
+	// Call ai request to solve the issues
+	if *flagAiApiProvider != "" && *flagAiApiKey != "" {
+		err := proposeSolution.GenerateSolution(*flagAiApiProvider, *flagAiApiKey, issues)
+		if err != nil {
+			logger.Fatal(err)
```

Comment:
I would just log this as a warning not fatal, because it will prevent the printing of the results when the autofix cannot generated by the AI. This fix is nice to have but not mandatory for getting the results of the scan.